CodeReview/src/CodeReviewManager.h

```diff
@@ -28,19 +28,41 @@
 
 #include "codereview_api.h"
 
+#include "VersionControlUI/src/nodes/DiffFrame.h"
+
 #include "nodes/CommentedNode.h"
 
 namespace CodeReview {
 
+using DiffFrames = QList<VersionControlUI::DiffFrame*>;
+
+using OrderedDiffFrames = QList<VersionControlUI::DiffFrame*>;
```

Comment:
We did? I do not remember that particular discussion. Perhaps there was a misunderstanding. When I talked about different types, I meant the names of the individual strategies.

I don't feel like naming a `QList<DiffFrame*>` in a few different ways makes any difference here. Do you think it helps in some way?